### ISSUE TITLE
Report for items sold and inventory

### ISSUE THREAD

**Issue by AUDevz** [nexopos/v3-issues#251]
**Username**:  AUDev

**Is your feature request related to a problem? Please describe.**
There is no reports for items sold and inventory, so there is no way of knowing how many items have been sold today and where does the inventory stand.
The inventory tracking sheet in reports only display categories which is useless as each category has multiple items and can't tell which item was sold.

**Describe the solution you'd like**
There should be a Report for items sold and inventory i.e number of items sold in certain date and inventory sheet that shows how many items are of products what price is of the products and total where inventory stands.
Basically the same inventory tracking sheet in reports only that it displays actual items, not categories which is useless.

**Describe alternatives you've considered**
Change the inventory tracking sheet and add a button which may switch between categories and items. or add a new report.

**Additional context (optional)**

**Comment by Blair2004** [comment]
Thank you for this, we'll consider checking that in details.

**Comment by Blair2004** [comment]
There is an upcoming module coming up that bring stock room module. as the module is at the first release, we'll be open to improve it.

https://www.youtube.com/watch?v=4sLBt2xf3qM&feature=youtu.be
